charts/zammad-ldap-sync/values.yaml

```diff
@@ -13,16 +13,32 @@ ldapSync:
         profiles:
       # Profile influences log output. May be you don't want to write ldap entries in your log files.
           active: default,prod
+      # Send email in caseof invalid distinguished-names.
+        mail:
+          host: test.host.com
```

Comment:
Please use mail.example.com otherwise host.com will get a lot of mails

Reply:
✅